frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/ExpressionEditorTextfield.tsx

```diff
@@ -40,6 +45,50 @@ import {
 ace.config.set("basePath", "/assets/ui/");
 ace.config.set("useStrictCSP", true);
 
+export type SuggestionFooter = {
+  footer: true;
+  name: string;
+  icon: IconName;
+  href: string;
+};
+
+type SuggestWithFooters = {
+  suggestions: (Suggestion | SuggestionFooter)[];
+  helpText?: HelpText;
+};
+
+function suggestWithFooters(
+  args: SuggestArgs,
+  showMetabaseLinks: boolean,
```

Comment:
Calls to this function get a bit awkward with 2 arguments.
WDYT about this?

```suggestion
  args: SuggestArgs & { showMetabaseLinks: boolean }
```